src/main/webapp/stats.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
Add a screenshot to your PR (usually only add it if you edit html). Also rename your PR to have a meaningful title. Here is an example that Kendall wrote previously https://github.com/FluffySheep/CodeU2019-Team28/pull/27